### ISSUE TITLE
Without purchasing any standard plan i can use the icon.

### ISSUE THREAD

**Issue by pinalp887** [FortAwesome/Font-Awesome#15164]

**Comment by AdrianoCahete** [comment]
[Of course not](https://fontawesome.com/license). But you can still use the free option.

**Comment by robmadole** [comment]
@pinalp887 are you saying that one of our services is allowing you to use a Pro icon even though you haven’t paid for it? If you think this is a security issue please send us an email at hello@fontawesome.com.

**Comment by pinalp887** [comment]
Yes, I can use the pro-icon without purchasing any standards plan.
@robmadole  I am sharing the details in the email please check it out.
